code/modules/atmospherics/machinery/components/fusion/hfr_parts.dm

```diff
@@ -40,10 +40,49 @@
 			return
 	if(default_change_direction_wrench(user, I))
 		return
-	if(default_deconstruction_crowbar(I))
-		return
 	return ..()
 
+/obj/machinery/atmospherics/components/unary/hypertorus/crowbar_act(mob/user, obj/item/tool)
+	if(!panel_open)
+		balloon_alert(user, "open panel!")
+		return TOOL_ACT_TOOLTYPE_SUCCESS
+
+	var/unsafe_wrenching = FALSE
+	var/filled_pipe = FALSE
+	var/datum/gas_mixture/env_air = loc.return_air()
+	var/datum/gas_mixture/ins_air = airs[1]
+	var/internal_pressure = 0
+	if(istype(src, /obj/machinery/atmospherics/components/unary/hypertorus/core))
+		var/obj/machinery/atmospherics/components/unary/hypertorus/core/core_part = src
+		internal_pressure = max(core_part.internal_fusion.return_pressure(), core_part.moderator_internal.return_pressure())
+		if(internal_pressure)
+			filled_pipe = TRUE
+
+	if(!nodes[1])
+		internal_pressure = max(internal_pressure, ins_air.return_pressure())	
+	
+	internal_pressure -= env_air.return_pressure()
+
+	if(ins_air.total_moles() > 0)
+		filled_pipe = TRUE
+
+	default_deconstruction_crowbar(tool, custom_deconstruct = filled_pipe)
+		
+	to_chat(user, span_notice("You begin to unfasten \the [src]..."))
+	
+	if(internal_pressure > 2 * ONE_ATMOSPHERE)
+		to_chat(user, span_warning("As you begin deconstructing \the [src] a gush of air blows in your face... maybe you should reconsider?"))
```

Comment:
This should be a balloon alert, though I know the others aren't. This should either be in the same balloon alert as "unfastening...", or sent after a very short delay.